README.md

```diff
@@ -86,37 +133,72 @@ Because we have not implemented a CI/CD pipeline with a self-hosted agent, we ne
 
 Run the following to:
 
-* Allow public access from your IP address, g
-* Give the logged in user permissions to upload a blob
-* Create the `deploy` container
-* Upload the zip file `./website/SimpleWebApp/SimpleWebApp.zip` to the `deploy` container
-* Tell the web app to restart
+- Allow public access from your IP address, g
+- Give the logged in user permissions to upload a blob
+- Create the `deploy` container
+- Upload the zip file `./website/SimpleWebApp/SimpleWebApp.zip` to the `deploy` container
+- Tell the web app to restart
 
 ```bash
-CLIENT_IP_ADDRESS=<your-ip-address>
+CLIENT_IP_ADDRESS=<your-public-ip-address>
 
-STORAGE_ACCOUNT_PREFIX=stg
+STORAGE_ACCOUNT_PREFIX=st
 WEB_APP_PREFIX=app-
 NAME_OF_WEST_STORAGE_ACCOUNT="$STORAGE_ACCOUNT_PREFIX$BASE_NAME"

```

Comment:
Can we rename this variable to remove WEST from the name? Maybe just NAME_OF_STORAGE_ACCOUNT